### DISCUSSION TITLE
The tanstack team is considering launching a new product. Can it support signals?

### DISCUSSION THREAD

**Discussion by 1057105012** [TanStack/router#2743] [General]
Design new state management, refer to useRef of vue, but compatible with react, used to solve several big problems of react
1. Fine-grained rendering problem
2. Re-rendering problem
3. When operations involving multiple immutable objects are involved, since the latest data cannot be obtained immediately, immer operations must be performed, which is very troublesome.

I think this feature is extremely important for the react ecosystem, but the react team doesn't seem to care about it.

**Comment by schiller-manuel** [comment]
no idea what this is about. please rephrase this.
